compiler/rustc_metadata/src/rmeta/encoder.rs

```diff
@@ -1595,6 +1595,11 @@ impl<'a, 'tcx> EncodeContext<'a, 'tcx> {
                 f.did.index
             }));
 
+            for field in &variant.fields {
+                // FIXME: this probably blows up rmeta size.
```

Comment:
You can go ahead and remove this FIXME. As I wrote [here](https://github.com/rust-lang/rust/pull/132915/files#r1837080093), making safety a defaulted table is the right initial decision in the absence of a good reason to do otherwise.